Runtime/DotNetE2EServerApp/Controllers/Table/AuthenticatedController.cs

```diff
@@ -19,9 +21,10 @@ public class AuthenticatedController : PermissionTableControllerBase
     {
         public override async Task<IQueryable<TestUser>> GetAll()
         {
-            MobileAppUser user = (MobileAppUser)this.User;
-            var creds = await user.GetIdentityAsync<FacebookCredentials>();
-            var all = (await base.GetAll()).Where(p => p.UserId == user.Id).ToArray();
+            ClaimsPrincipal user = this.User as ClaimsPrincipal;
+
+            var creds = await user.GetAppServiceIdentityAsync<FacebookCredentials>(this.Request);

```

Comment:
fyi - this GetAll function I've rewritten on my side so it does something closer to what it was doing.